2024-03-27/minutes.md

```diff
@@ -1 +1,431 @@
+
+So some stuff is still going to have to stay on repositories, READMEs, CONTRIBUTING.md, things like that.
+
+[Onkar]
+We can have minimal stuff on the repository and most of the checking out, the git stuff, all the small small things about using it.
+
+[Ruth John]
+I think there is some "how to use GitHub" stuff in both places, we should review it again.
+We can review the contributing MD on content that we link to, revise that and make sure that when people do click on Contributing they're taken still to MDN rather than back and forth.
+
+[Estelle]
+The other thing we can do is since it's all markdown; point to the blob on GitHub which is actually then automatically rendered so it's just written in one place.
+And where you link to it if you're in GitHub you can link to it from GitHub like from the README file.
+
+[Ruth John]
+We don't want to duplicate content and have to keep it up to date in 2 places.
+So that does help solve that problem, definitely.
+Unless you have macros. I think we can avoid them in our contributing docs.
+
+[Estelle]
+I mean, there'll be the sidebar macro, but that would be about it. Onside.
+
+[Ruth John]
+Yeah, I think that's a blocker. As far as some of the suggestions on where this link should go, I think we carry that on in the discussion.
+Some of the suggestions already in there crossing over with some work that Anuja, our UX designer is currently doing on site-wide navigation.
+But things like adding Writing Guidelines specifically unobtrusively is okay.
+I think if we have a markdown page that links to all the different things on MDN, then we can harness that and we can link to that.
+
+[Brian Smith]
+Anything else? Anyone?
+
+[Onkar]
+No, just contribute to the discussions. If you put your comments over there, then we know that it is making progress.
+
+[Estelle]
+I guess what you're also saying is we should pull out Glossary from the Writing Guidelines discussion and create a separate discussion for Glossary.
+
+[Ruth John]
+I think maybe that's a good idea. Cause I think that user flow for the guidelines and where it comes from where it goes to and everything is probably gonna take over from Glossary discussion.
+There'll be 2 conversations happening at the same time.
+
+[Onkar]
+Okay, I'll create a separate discussion.
+
+[Ruth John]
+Thanks.
+
+[Estelle]
+Thank you for all your contributions Onkar
+
+[Brian Smith]
+Yeah, absolutely.
+
+[Onkar]
+My pleasure.
+
+## Closing discussions
+
+[Brian Smith]
+If there's anything else that you want to discuss, talk about, bring up. Now's a good time.
+
+[Estelle]
+Just a section on the discussions, which is I went through a bunch of discussions. And. So if I hit you up asking you a question about, moving your discussion forward.
+There's a lot of discussions that have been opened since like 2022 so don't take it personally if I ask you if you Want to move forward to closer because some of them.
```

Comment:
```suggestion
I went through a bunch of discussions. So I might have hit you up asking you a question to move your discussion forward. There's a lot of discussions that have been opened since like 2022.
```